news/2021-05-13-mappers-report-april.md

```diff
@@ -0,0 +1,172 @@
+
+| Award | Team | Submission |
+| :-- | :-- | :-- |
+| ![A Labour of Love: Community Pick](https://assets.ppy.sh/profile-badges/alol_communitypick.png "A Labour of Love: Community Pick") | Team Red but Blue | [Kuba Oms - My Love (Redside Remix) (feat. Hatsune Miku)](https://osu.ppy.sh/beatmapsets/1388993) |
+| ![A Labour of Love: Overall Runner-up](https://assets.ppy.sh/profile-badges/alol_overallrunnerup.png "A Labour of Love: Overall Runner-up") | schoolboy fans | [Kuba Oms - My Love (Shadren Remix)](https://osu.ppy.sh/beatmapsets/1382412) |
+| ![A Labour of Love: Overall Winner](https://assets.ppy.sh/profile-badges/alol_overallwinner.png "A Labour of Love: Overall Winner") | extremely valid | [Raphlesia & BilliumMoto - My Love](https://osu.ppy.sh/beatmapsets/1388906) |
+
+Details regarding the results, along with the livestream VOD (which includes commentary by pishifat, Uberzolik and Ephemeral) can be viewed in [this newspost](https://osu.ppy.sh/home/news/2021-04-27-results-a-labour-of-love). For further information about the contest, we recommend you pay a visit to the [wiki article](/wiki/Contests/A_Labour_of_Love_Contest)!
+
+### Monthly Beatmapping Contest: March & April
+
+Just like our monthly mapping-flavoured bulletin, the streak continues for the Monthly Beatmapping Contest as well.
+
+If you recall, the March edition of the contest was held in both osu! and osu!taiko game modes. On the osu! side of things, we have an underdog among the veterans, [Innovation](https://osu.ppy.sh/users/6304412) winning the contest.
+
+Appearing as the runner-up, [Mao](https://osu.ppy.sh/users/2204515), on the other hand, not only achieves the third tier profile badge but also attains the *Elite Mapper* user title, which later turns into *Elite Mapper II* thanks to his participation in the [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) winning entry. As for the Screener Crew #1, [Acylica](https://osu.ppy.sh/users/1943309) takes the spotlight and getting an inch closer to the *Elite Mapper* title. Huge applause to them for their flawless victories!
+
+osu!taiko section of the contest had [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) rise forth to the first place, followed by [Nifty](https://osu.ppy.sh/users/4956097) as the runner-up. Many congrats to both of them!
+
+If you're craving for some details, [this newspost](https://osu.ppy.sh/home/news/2021-04-23-results-monthly-beatmapping-contest-march-2021) should satisfy your needs.
+
+Next up, the April edition of Monthly Beatmapping Contest greets us with a few novelties. For the first time ever, the contest will entertain osu!catch submissions in addition to osu! and osu!taiko. Though the submission period has since ended, the limitations for April were as follows:
```

Comment:
```suggestion
Next up, the April edition of Monthly Beatmapping Contest greets us with a few novelties. For the first time ever, the contest will entertain osu!catch submissions in addition to osu! and osu!taiko. The submission period has since ended, but for those interested, the limitations for April were as follows:
```